sherpa/ui/tests/test_ui.py

```diff
@@ -311,8 +312,21 @@ def test_ui_source_methods_with_full_model(clean_ui, setup_ui_full):
                        match="Convolved model\n'powlaw1d.p1'\n is set for dataset full. You should use get_model_plot instead."):
         ui.get_source_plot('full')
 
-    # Test Case 2
-    ui.set_source('full', 'powlaw1d.p2')
+    # Test Case 2.
+    #
+    # The hide_logging fixture hides the logging output, so we need
+    # to re-enable it for this call.
+    #
```

Comment:
I do note the that `hide_logging` docstring says, "ignore info", but then sets the lefel to ERROR (meaning it hides INFO *and* WARNING). If `hide_logging` did what it's advertizes as, then we would not have to reset that here, because this is WARNING level. 
However, if you want, we can leave as is - this is just a test where 100% consistency is not required; we are allowed to do inconsistent setups as it makes sense for each test.

Reply:
In this particular case I think it useful that we run this test at the INFO level, so we can check what we create for the "standard" user experience.

I don't have the energy/desire to fix up the hide_logging docstring to point out it hides warning too.

Reply:
I'm not planning to change this unless required - see below (as I didn't want the comment to get lost with future changes/rebases) https://github.com/sherpa/sherpa/pull/1800#issuecomment-1599268830

Reply:
I understand. That's why I approved this yesterday.